predicators/approaches/maple_q_approach.py

```diff
@@ -124,7 +124,7 @@ def _learn_nsrts(self, trajectories: List[LowLevelTrajectory],
                 assert nsrt.option_vars == nsrt.parameters  # pragma: no cover.
         # On the first cycle, we need to register the ground NSRTs, goals, and
         # objects in the Q function so that it can define its inputs.
-        if not online_learning_cycle:
+        if not online_learning_cycle and CFG.approach != "rl_bridge_policy":
```

Comment:
can you adda  comment here explaining why we need ` CFG.approach != "rl_bridge_policy"`?